src/pageObject.js

```diff
@@ -1,16 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
```

Comment:
I would add a blank line between the methods
```suggestion
  constructor(url) {
    this.url = url;
    this.header = new Header();
  }
  
  clickOnLogo() {
    return `Click on the ${this.header.logo}`;
  }
```